src/components/Wizard/Stories/WizardPatternExample.js

```diff
@@ -148,22 +163,21 @@ export class WizardPatternExample extends React.Component {
           steps={wizardSteps}
           loadingTitle="Loading..."
           loadingMessage="This may take a minute."
-          loading={this.state.wizardLoading}
-          activeStepIndex={this.state.activeStepIndex}
           onStepChanged={activeStepIndex => this.setState({ activeStepIndex })}
+          {...knobs}
         />
       </div>
     );
   }
   renderStateful() {
+    // No knobs for the stateful example, we want to let it control its own state.
```

Comment:
i was more or less showing that derived state is also overridable but i'm fine with this convention as well ;)

Reply:
Oh, that's true... hmm! Maybe worth having the knobs in both cases. But then the storybook stories look identical, haha! hmm.